hw5.rb

```diff
@@ -0,0 +1,26 @@
+puts '#2'
+names.each { |el| print el.length }
+puts
+puts '#3'
+names.each { |el| print el.downcase }
+puts
+puts'#4'
+puts h=Hash[names.map{|el| [el, el.size]}]
+puts'#5'
+puts st1=h.invert.max&.last
+puts'#6'
+puts  names.index(st1)
+puts'#7'
+puts names.sort_by(&:length)
+puts names.sort_by(&:length).reverse
```

Comment:
Print an array, beginning with the longest name from `names` (from `names.index(st1)` ) until the last element of `names`